lib/gui/etcher.js

```diff
@@ -35,8 +35,13 @@ electron.app.on('before-quit', () => {
 })
 
 electron.app.on('ready', () => {
-  // No menu bar
-  electron.Menu.setApplicationMenu(null)
+
+  if (process.platform === 'win32') {
```

Comment:
Why specifically different behaviour on Windows?

Reply:
Because the menu is displayed within the window on Windows, which we don't want – but I still need to check if the shortcuts etc. still work there.